notifier/orb.yaml

```diff
@@ -1,12 +1,56 @@
 version: 2.1
 description: "Tools for notifying external services about CI events"
 
+orbs:
+  slack: circleci/slack@4.7.0
+
 executors:
   alpine-stable:
     docker:
     - image: alpine:latest
     resource_class: small
 
+commands:
+  slack-on-failure:
```

Comment:
We already have a job named `slack`. Perhaps with the addition of this new command it would make more sense for that job to be named `slack-on-deploy` or similar?

Reply:
You are right! I changed `slack-on-failure` for `slack-on-deploy`.

Reply:
Oh I meant changing our old `slack` job to `slack-on-deploy` 😄 so we'd have the `slack-on-deploy` job and the `slack-on-failure` command

Reply:
That (second comment) makes sense to me! Note that would definitely be a breaking change, so we'll need to do a major version bump (see the [semver spec](https://semver.org/), which we try to stick to everywhere)

Reply:
Changing `slack` for `slack-on-deploy` requires changing all the uses of this job on the other repos, right?
Which repos should I change in addition to realtime-nlp, realtime-finalizer and realtime-decoder?

Reply:
Looks like we use it in a bunch of our repos:
```
~/s/talkiq » rg -c 'notifier/slack' $(fd -uuu config.yml)
./realtime-ctm/.circleci/config.yml:3
./realtime-finalizer/.circleci/config.yml:3
./realtime-egress/.circleci/config.yml:3
./realtime-registration/.circleci/config.yml:3
./redis/.circleci/config.yml:2
./autoscaler/.circleci/config.yml:3
./realtime-nlp/.circleci/config.yml:3
./realtime-decoder/.circleci/config.yml:3
./offline/.circleci/config.yml:3
./realtime-replicator/.circleci/config.yml:3
./gc-config/.circleci/config.yml:1
```

Reply:
Ok, I will do 11 PRs (one per each repo) changing `slack` for `slack-on-deploy`

Reply:
@leanaha note that once you release this orb as a new version, Renovate should automatically open PRs on each repo to do the update. You should be able to piggy-back off those PRs to get a starting point :)

Reply:
Thanks! I was thinking of making those PRs manually.